doc/security/security-best-practice-en.md

```diff
@@ -0,0 +1,83 @@
+# Fate security best practice guidance
+
+When the FATE is used to perform tasks such as federated learning, security models and security protocols should be used to ensure that no privacy data is disclosed during the calculation process. This draft attempts to provide security instructions for using FATE.
+## 1. State of Art of FATE
+
+FATE includes multiple federal learning protocols, some of which are composed of participants and third-party arbiters, such as Homo LR. Other federated learning protocols do not require third-party arbiter to do parameter aggregation, such as secureboost, Hetero Neural Network, Hetero FTL, etc. Therefore, FATE security model can be divided into two types: with third-party arbiter and without third-party arbiter.
+
+For the federated learning without third-party arbiter, its security assumption is: each participant in FATE is an honest, curious and independent security domain, and the data interaction between participants can ensure that the privacy data of each security domain is not leaked. For the federated learning with third-party arbiter, the security assumption is: the federal learning participants are honest, curious and independent security domains, and the arbiter is honest and curious for all participants. That is, the arbiter will comply with the FATE protocol's execution, but may retain or inference aggregation stage intermediate data. When multiple arbiters perform aggregation and arbitration operations, they must not collude with each other.
```

Comment:
"honest, curious" -> honest-but-curious.
"privacy data" -> private data.
And not sure what are "independent security domains"? Never see this definition before.

Reply:
"Independent security domain" means different participants in FATE have their own trust boundary. Except the security protocol, there is no other information exchange between these participants(domains). Using independent security domain could assess participants' security status and threat model more clearly.

Reply:
"Independent security domain" means different participants in FATE have their own trust boundary. Except the security protocol, there is no other information exchange between these participants(domains). Using independent security domain could assess participants' security status and threat model more clearly.